File: XML/XMLDom/ChuongTrinh/DocPhanSo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml;
using System.IO;

namespace XMLDom.ChuongTrinh
{
    internal class DocPhanSo
    {
        private int tuSo, mauSo;
        private string duongDan = Directory.GetCurrentDirectory() + @"..\..\..\DuLieu\PhanSo.xml";

        public void PhanSo()
        {
            XmlDocument xmlDocument = new XmlDocument();
            xmlDocument.Load(duongDan);

            XmlElement goc;
            goc = xmlDocument.DocumentElement;

            tuSo = int.Parse(goc.GetAttribute("Tu_So"));
            mauSo = int.Parse(goc.GetAttribute("Mau_So"));

            Console.WriteLine($"Phân số là: {tuSo}/{mauSo}");
        }
    }
}

File: XML/VB_XML/ChuongTrinh/DocPhanSo.vb
Imports System.Xml

Module DocPhanSo

    Public Sub Main()
        Dim TuSo, MauSo As Integer
        Dim DuongDan As String = "..\..\..\DuLieu\PhanSo.xml"
        Dim TaiLieu As New XmlDocument
        TaiLieu.Load(DuongDan)
        Dim Goc As XmlElement = TaiLieu.DocumentElement
        TuSo = Goc.GetAttribute("Tu_So")
        MauSo = Goc.GetAttribute("Mau_So")

        Dim Chuoi As String = "Phan so: "
        Chuoi &= TuSo & "/" & MauSo
        Console.WriteLine(Chuoi)
        Console.ReadLine()
    End Sub

End Module